lib/at_cmd_parser/at_cmd_parser.c

```diff
@@ -52,6 +64,22 @@ static inline void skip_command_prefix(const char **cmd)
 	(*cmd)++;
 }
 
+static inline bool check_response_for_forced_string(const char *tmpstr)
```

Comment:
If possible, avoid `inline` here. I don't think there is any reason to inline this or `is_clac` down below (other than similar functions in this file being `inline`).

Reply:
Fixed